Jenkinsfile

```diff
@@ -801,6 +797,7 @@ pipeline {
             }
             environment{
                 WORKAROUND_iGemm_936 = " MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_FWD_V4R1=0"
+                Navi21_build_cmd = "CTEST_PARALLEL_LEVEL=2 MIOPEN_CONV_PRECISE_ROCBLAS_TIMING=0 MIOPEN_LOG_LEVEL=5 make -j\$(nproc) check"
```

Comment:
I guess this should not be necessary anymore (or still needed)?

Reply:
@atamazov still testing under draft status, adjusting "replay" and change it to 4 some times. :)
soft_max on OCL platform failed last CI, it seems that this test has been problematic from another issue.

Reply:
Isee. Maybe soft_max should be RUN_SERIAL?

Reply:
```suggestion
                // WORKAROUND_ISSUE_1148:
                Navi21_build_cmd = "CTEST_PARALLEL_LEVEL=2 MIOPEN_CONV_PRECISE_ROCBLAS_TIMING=0 MIOPEN_LOG_LEVEL=5 make -j\$(nproc) check"
```